lib/rotoscope.rb

```diff
@@ -34,17 +34,16 @@ def self.trace(dest, blacklist: [], flatten: false)
   end
 
   def flatten(dest)
-    io_given = false
-    dest_file = if dest.respond_to?(:to_io)
-      io_given = true
-      dest.to_io
+    io_given = if dest.respond_to?(:puts)
```

Comment:
This method is part of the [public API](https://github.com/Shopify/rotoscope/blob/a1c557a6cc6a17cff85f1b97a239974b0c7e9bad/README.md#rotoscopeflattendest) and it seems convenient to support both a string of a filepath or an IO instance.